CHANGELOG.md

```diff
@@ -4,6 +4,45 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.0.0-rc.0
+
+This is the first release candidate of Sentry JavaScript SDK v8. Except for missing ESM support, this version is
+considered stable.
+
+We recommend to read the detailed [migration guide](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/) in
+the docs.
+
+### Version Support
+
+The Sentry JavaScript SDK v8 now supports Node.js 14.8.0 or higher. This applies to @sentry/node and all of our
+node-based server-side sdks (@sentry/nextjs, @sentry/remix, etc.).
+
+The browser SDKs now require ES2018+ compatible browsers. New minimum browser versions:
+
+Chrome 63 Edge 79 Safari/iOS Safari 12 Firefox 58 Opera 50 Samsung Internet 8.2
+
+For more details, please see the
+[version support section in migration guide](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#1-version-support-changes).
+
```

Comment:
```suggestion
```

I think we can omit this as well here then, keep it a "normal"/"small" changelog!